src/current/v23.1/configure-logs.md

```diff
@@ -254,9 +254,11 @@ Along with the [common sink parameters](#common-sink-parameters), each HTTP serv
 | `unsafe-tls`          | When `true`, bypasses TLS server validation.<br><br>**Default:** `false`                                                                                                                                                                                                                                                                                           |
 | `timeout`             | Timeout before requests are abandoned.<br><br>**Default:** `0` (no timeout)                                                                                                                                                                                                                                                                                        |
 | `disable-keep-alives` | When `true`, disallows reuse of the server connection across requests.<br><br>**Default:** `false` (reuses connections)                                                                                                                                                                                                                                            |
+| `compression` | Compression method for the HTTP request body. Valid values `gzip` or `none`.<br><br>**Default:** `gzip` Prior to v23.1.4, requests were not compressed by default. |
```

Comment:
Since you're mentioning such a specific point version, suggest linking to that version's release notes: https://www.cockroachlabs.com/docs/releases/v23.1#v23-1-4

Reply:
added link